.github/workflows/pr_push.yml

```diff
@@ -14,17 +14,17 @@ jobs:
   FastBuild:
     name: Fast build
     env:
-      HWLOC_PATH: "${{github.workspace}}/build/vcpkg/packages/hwloc_x64-windows"
+      VCPKG_PATH: "${{github.workspace}}/build/vcpkg/packages/hwloc_x64-windows;${{github.workspace}}/build/vcpkg/packages/tbb_x64-windows"
     strategy:
       matrix:
         include:
           - os: windows-latest
-            pool_scalable: 'OFF'
+            pool_scalable: 'ON'
```

Comment:
you enable pool_scalable in each matrix entry - could you just enable it in cmake options? This is a fast build so we don't need to disable it here

Reply:
Done